examples/app.py

```diff
@@ -55,3 +67,22 @@ def playlists():
         return redirect('/')
     else:
         return spotify.current_user_playlists()
+
+@app.route('/currently_playing')
+def currently_playing():
+    if not session.get('token_info'):
+        return redirect('/')
+    else:
+        track = spotify.current_user_playing_track()
+        if not track is None:
+            return track
+        else:
+            return "No track currently playing."
```

Comment:
This is equivalent to:

    return spotify.current_user_playing_track() or "No track currently playing."